<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="lmt_box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lopsided-Margins Test</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plan_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="schwartzberg_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Schwartzberg</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pp_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Polsby-Popper</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="eg_box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Efficiency Gap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plan_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="plan_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Browse to plan folder</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reock_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reock</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_hull_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Convex Hull Ratio</string>
   </property>
  </widget>
  <widget class="QLabel" name="shape_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select shapefile (if present)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shape_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shape_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="compare_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Compare Plans</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mmd_box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mean-Median Difference</string>
   </property>
  </widget>
  <widget class="QLabel" name="test_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose tests</string>
   </property>
  </widget>
  <widget class="QLabel" name="historic_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select historic election data (if present)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="historic_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="historic_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>